src/webapi/blob.rs

```diff
@@ -31,6 +34,46 @@ pub trait IBlob: ReferenceType {
             Some( mime )
         }
     }
+
+    /// Create a new `Blob` object containing the data in the specified range of bytes of the
+    /// source `Blob`.
+    /// 
+    /// See also [slice_with_content_type](IBlob::slice_with_content_type).
+    /// 
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Blob/slice)
+    // https://w3c.github.io/FileAPI/#ref-for-dfn-slice
+    fn slice< T >( &self, range: T ) -> Blob
+        where T: RangeBounds<u64>
+    {
+        self._slice(range, None)
+    }
+
+    /// [slice](IBlob::slice) `Blob` with the provided `content_type`.
+    fn slice_with_content_type< T >( &self, range: T, content_type: &str ) -> Blob
```

Comment:
`IBlob` also has a function named `mime` of which the meaning is the same as `content_type` here. Maybe I need to change either so that the two to be consistent? Any idea?

Reply:
It's just a name so it doesn't really matter that much. We can't change the name of the `mime` method without breaking backwards compatibility, so the only thing we can do is to change the name of this parameter. You can rename it to `mime` if you want.